cdci_data_analysis/pytest_fixtures.py

```diff
@@ -694,6 +696,39 @@ def empty_products_user_files_fixture(default_params_dict, default_token_payload
 
     yield scratch_params
 
+@pytest.fixture
+def dispatcher_parameterized_hard_minimum_age_days(pytestconfig, dispatcher_test_conf_fn, dispatcher_debug, request):
```

Comment:
could we doing without duplicating code?

Reply:
That's what I want. But I am a bit confused on how to pass the parametrized configuration (hard limit parameter), to the dispatcher_live_fixture.

Any ideas?

Reply:
I would try to make a parametrized variation of `dispatcher_test_conf_fn`, perhaps inheriting from this one, or straight modifying, with an empty default. It takes some attempts.
Maybe to save time now we could have a way without changing config, just changing the files.

Reply:
I parametrized `dispatcher_live_fixture`. but It's not the best solution for sure,

Reply:
I imagine it can cause issues. But there is a todo. If it works it's fine

Reply:
what I was trying to achieve is a bit convoluted and it probably requires quite more time, but it would be cool to have it working in the future